tests/test_ops/test_nms.py

```diff
@@ -23,10 +23,23 @@ def test_nms_allclose(self):
         assert np.allclose(dets, np_dets)  # test cpu
         assert np.allclose(inds, np_inds)  # test cpu
         dets, inds = nms(
-            boxes.cuda(), scores.cuda(), iou_threshold=0.3, offset=0)
+            boxes.to(device), scores.to(device), iou_threshold=0.3, offset=0)
         assert np.allclose(dets.cpu().numpy(), np_dets)  # test gpu
         assert np.allclose(inds.cpu().numpy(), np_inds)  # test gpu
 
+    @pytest.mark.parametrize('device', [
+        pytest.param(
+            'cuda',
+            marks=pytest.mark.skipif(
+                not is_cuda(), reason='requires CUDA support')),
+        pytest.param(
+            'mlu',
+            marks=pytest.mark.skipif(
+                not is_mlu(), reason='requires MLU support'))
+    ])
+    def test_nms_allclose(self, device):
```

Comment:
Why do we need to move those lines to `_test_nms_allclose`?

Reply:
I refered the focalloss OP before, but I think we can only reserve the `test_nmsallclose` function. Shall we?

Reply:
Yet

Reply:
OK, I have fixed it.